### ISSUE TITLE
Navbar flickering

### ISSUE THREAD

**Issue by wagstorm** [Dogfalo/materialize#754]
Hi. I was having problem with the navbar flickering. So Dogfalo suggested to put ``` transform: translateZ(0); ``` in the code - and it worked (I did use vendor prefixes for transform to do this on various browsers). But then another bug occurred: In chrome (odly not in IE) the mobile navbar disappeared after the menu items stopped.

![image](https://cloud.githubusercontent.com/assets/4126339/6379577/a7d8d208-bd33-11e4-8f14-c40da333e0b8.png)

If I set translateZ(1); it fixes the Chrome problem, but the top navbar sometimes disappear when scrolling again.

**Comment by acburst** [comment]
Please give us more information preferably code, and how to replicate the bug

**Comment by troendheim** [comment]
Okay, the problem occurs whenever this is set: 
`nav {
    transform: translateZ(0);
    -webkit-transform: translateZ(0); /* plus the other browsers */
}`

If translateZ is not defined, there is no problem, however the navbar flickers in some cases.

The navbar itself is defined like this: 
```html
<div class="navbar-fixed">
    <nav class="teal lighten-2">
        <div class="nav-wrapper container text-black">
            <a href="#" class="brand-logo"><span class="white-text">Logo</span></a>
            <a href="#" data-activates="mobile-sidenav" class="button-collapse"><i class="mdi-navigation-menu"></i></a>
            <ul id="nav-mobile" class="right hide-on-med-and-down">
                <li><a href="find-lejlighed.html" ><span class="white-text">Find lejlighed</span></a></li>
                <li><a href="udlej-lejlighed.html"><span class="white-text">Udlej lejlighed</span></a></li>
            </ul>
            <!-- Remember to change this corresponding to the nav just above -->
            <ul class="side-nav" id="mobile-sidenav">
                <li><a href="find-lejlighed.html" ><span>Find lejlighed</span></a></li>
                <li><a href="udlej-lejlighed.html"><span>Udlej lejlighed</span></a></li>
            </ul>
        </div>
    </nav>
</div>
```

I tried to narrow it down my selectively excluding parts of the code so that they would not interfere, but I didn't find anything.

I'm not sure if you gained anything from this. I'll return if with any findings I have.

**Comment by wagstorm** [comment]
The flickering happens in: Chrome Version 40.0.2214.115 m

**Comment by nathanmarcos** [comment]
I have the same problem on Chrome Version 41.0.2272.89 m.

**Comment by feliperdius** [comment]
The same problem here. 
See here: http://codepen.io/anon/pen/PwXdPP

**Comment by nathanmarcos** [comment]
Nice sample @feliperdius, exactly the same problem here.

**Comment by TheAdamGalloway** [comment]
Also having this problem. 
Firstly, Internet Explorer completely ruins my static navbar.
Secondly, Chrome refuses to display the mobile menu over any content other than the navbar.

Unsurprising, Firefox does everything perfectly.

**Comment by atomala** [comment]
Having the same problem as everyone else is having here on Chrome 41.0.2272.118 m

**Comment by TheAdamGalloway** [comment]
Have you tried upgrading to the latest Materialize? Fixes it for me...

**Comment by Dogfalo** [comment]
This is fixed on the latest version v0.96.1
